ts/Extensions/PriceIndication.ts

```diff
@@ -372,14 +373,25 @@ addEvent(Series, 'afterRender', function (): void {
         }
 
         if (lastVisiblePrice && lastVisiblePrice.enabled && pLength > 0) {
-            crop = (points[pLength - 1].x === x) || pointRange === null ? 1 : 2;
```

Comment:
A few lines above you can remove `pointRange` definition (line 311)

We need to clean up the leftovers manually (it is disabled in eslinter afaik), sorry

Reply:
PS: What happens when `pointRange === null` in your solution? Does it even matter?

Reply:
That reminds me of `cropShoulder` property. Could you check this for me? 
`cropShoulder = 1`  for line-based series and `cropShoulder = 0` for column-based series. That property defines how many extra points are rendered. My guess is we should use this value to determine the "last point" when data is cropped (`series.cropped` property). That would be faster than looping over all points.

I mean, otherwise instead of looping over, why won't we simply do this:
```ts
const lastPoint = points[length - 1].isInside ? points[length - 1] : points[length - 2]
```

Reply:
I've tested the `pointRange` property while implementing this solution, and it seems like everything works fine with the `pointRange === null` and with other numeric values.